proprietary/Components/src/utrecht/form-field-description.tokens.json

```diff
@@ -0,0 +1,12 @@
+{
+  "utrecht": {
+    "form-field": {
+      "description": {
```

Comment:
It is `"form-field-description": {` instead of in two levels.

See: https://github.com/nl-design-system/utrecht/blob/main/components/form-field-description/tokens.json#L3